docs/topics/upgrade.md

```diff
@@ -100,3 +100,24 @@ There are known limitations with VMSS cluster-autoscaler scenarios and upgrade.
 ### Cluster-autoscaler + Availability Set
 
 We don't recommend using `aks-engine upgrade` on clusters that have Availability Set (non-VMSS) agent pools `cluster-autoscaler` at this time.
+
+## Forcing an upgrade
+
+The upgrade operation takes an optional `--force` argument :
+
+```shellscript
+-f, --force   
+force upgrading the cluster to desired version. Allows same version upgrades and downgrades.
+```
+
+In some situation, you might want to bypass the aks-engine validation on your api-model versions and cluster nodes versions. This is at your own risks and you should assess the potential harm of using this flag.
+
+The `--force` parameter instructs the upgrade process to :
+
+- ignore normal version validation
```

Comment:
```suggestion
- bypass the usual version validation.
```

Reply:
Actually, even normal works. But I still recommend the rest of the changes in my GH suggestion.